j4-dmenu-desktop.1

```diff
@@ -76,7 +76,8 @@ Performing
 will exit the program.
 .It Fl Fl wrapper Ar wrapper
 A wrapper binary.
-Useful in case you want to wrap into 'i3 exec'.
+Using to wrap into 'i3 exec' is deprecated. See
+.Fl Fl i3-ipc .
```

Comment:
I am not a native English speaker, but this doesn't look like valid English to me. Please also validate manpages with `man -T lint ./j4-dmenu-desktop.1` (this works with `mdocml`, I don't know if it works for `man-db`). Sentences need to be separated by a newline.

The change could look something like this (inspired by `--skip-i3-exec-check`'s description):

```suggestion
Usage of
.Ql --wrapper Qq i3 exec
and
.Ql --wrapper Qq sway exec
is deprecated, use
.Fl Fl i3-ipc
instead.
```